runtime/lua/vim/lsp/util.lua

```diff
@@ -1730,8 +1730,17 @@ end)
 ---
 ---@param locations table list of `Location`s or `LocationLink`s
 ---@param offset_encoding string offset_encoding for locations utf-8|utf-16|utf-32
+---@param opts (table, optional) Holds the following optional fields:
+---     - format (function|nil): Can be used to customize the `text` field 
+---       of the generated quickfix item. Takes the contents of the line
```

Comment:
```suggestion
---     - format (function|nil): Formats the quickfix item `text` field. Function args are ...
```

also show a terse example of the function signature